### DISCUSSION TITLE
Workflow turkeyGitops/ build failure

### DISCUSSION THREAD

**Discussion by Adi-FHE** [mozilla/hubs#5429] [Hubs Cloud Questions] [answered]
I am currently trying to create new branch in forked repository of mozilla/hubs. This branch would be merged with `(mozilla/hubs) upstream/hubs-cloud` and `(forked) origin/my-customHubs (Our custom feature)`. Whenever I pushed my local changes in newly created branch into origin (forked) remote with `git push origin mybranch `, following docker login error` (workflow error in turkeyGitops/build)` ensue. I still can do access and manipulate the branch in my forked repository and the local hubs client also running as expected. My only concern is this workflow Error, which discourage me from uploading the branch to my live server. If anyone kind enough to enlighten or point me to the right direction on why this problem occurs, i would really thankful.
![bd6f54ddab675ae7ab0cfd84c6e12029](https://user-images.githubusercontent.com/32239509/168259175-03883782-4f45-451d-a2ca-42c6e1a462ea.png)

**Comment by pattersonbl2** [comment, marked as answer]
I wouldn't worry about this error, it's a workflow for a cicd pipeline.

**Reply by Adi-FHE** [reply]
Would it be save to remove this workflow entirely from my custom client?

**Reply by pattersonbl2** [reply]
it would be safe to remove on your fork.

**Reply by Adi-FHE** [reply]
Thank you very much kind sir.
